packages/opencensus-core/src/stats/model/stats-manager.ts

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as logger from '../../common/console-logger';
+import * as loggerTypes from '../../common/types';
+import {ConsoleStatsExporter} from '../../exporters/console-exporter';
+import {StatsExporter} from '../../exporters/types';
+
+import {StatsAggregation} from './aggregation';
+import {Aggregation, BucketBoundaries, Measure, Measurement, RegistredViews, Stats, View} from './types';
+
+type RegistredMeasures = {
+  [key: string]: Measure;
+};
+
+export class StatsManager implements Stats {
```

Comment:
Is it necessary to define a `Stats` interface at all, if this is the only known implementation?

Reply:
The interface was so that users could create new implementations from it. But if you feel better I can remove it.